src/python/T0/RunConfig/Tier0Config.py

```diff
@@ -573,6 +573,18 @@ def setProcessingSite(config, site):
     config.Global.ProcessingSite = site
     return
 
+def setStreamMem(config, streamname, **options):
+    """
+    _SetStreamMem_
+    Define memory usage for any type of streamer configurable.
+    The following keys may be passed:
+        value - size of memory to be assigned 
+    """
+    if 'value' in options:
```

Comment:
I don't understand why the method receives an unespecified number of optional values. Do we need something other than 'value' ? is there a use case in which we wouldn't provide 'value'?. I think we could just use:
```
def setStreamMem(config, streamname, value):
```
or expand the scope of the funciont and make it a tool to define the full streamer configuration:
```
def addStreamerConfig(config, streamname, value):
```